ui/app/components/pending-tx/confirm-send-ether.js

```diff
@@ -148,7 +149,7 @@ ConfirmSendEther.prototype.getData = function () {
       address: txParams.to,
       name: identities[txParams.to] ? identities[txParams.to].name : 'New Recipient',
     },
-    memo: txParams.memo || '',
+    memo: hexToAsciiString(txParams.data) || '',
```

Comment:
@kumavis This is relying on turning memo into hex values so that it persists as tx data. I recall that you suggest using indexDB or localStorage to store memos instead of using txData. Is it because we don't want to pollute txData? 

p.s. This also wouldn't work for sending token.

Reply:
we have to strip txParam
and that we need to keep it clean
for signing
 ```js
txMeta: {
// stuff that metamask needs t know about it
  txParams: {
    //the real tx that we sign and send to the network
  }
}
```
thats the pattern we keep and txParams are not a good place to write metamask things…